### PR TITLE
fix(repo-name): fix whole repo structure for releasing with the new name

### PR DESCRIPTION
**Pull request by lucasfcosta**
This fixes the whole repo structure including the name of built files, tests and docs.
Please only merge this after renaming this repo to `get-func-name` in order to keep things consistent.

### CODE REVIEW

**Comment by shvaikalesh** [comment]
Maybe we should change [name of exported function](https://github.com/chaijs/get-function-name/blob/master/index.js#L23) too?

**Comment by keithamus** [comment]
Agreed. Let's change that one line and then this'll get my approval

**Comment by vieiralucas** [comment]
IMHO the exported function's name is correct because it still represents what that function does, the module's name is something different.

We can do whatever you want, I don't have a strong opinion on that.
This is just my 2cents on this matter.

@meeber can you give us your opinion here? We need you to untie this 😄

**Comment by lucasfcosta** [comment]
I agree with @vieiralucas, I think it's not a big deal either and we can do whatever you guys think it's best, but the exported function's name doesn't necessarily has to represent the module's name (even though they mean the same thing). The module's name has just been changed so we can get that namespace on NPM, but it is still more semantically correct (given what that function does) to call it `getFunctionName`.

Please let me know if you guys still disagree, thanks everyone for sharing your thoughts 😄

**Comment by shvaikalesh** [comment]
I do not have strong opinion on this, `func` vs `function`, kinda the same thing, but `func` is consistent with pkg name. I am happy with any name.

**Comment by meeber** [comment]
I vote `retrieveTheNameOfTheProvidedFunctionNoMatterWhatTypeOfFunctionItIsOrWhatTheCurrentJavaScriptEnvironmentIs`.

Second choice is to keep it consistent with `getfuncName`.

**Comment by lucasfcosta** [comment]
Here you go!
Also, sorry for the long discussion on this one hahaha 😄

**Comment by shvaikalesh** [comment]
LGTM, great work @lucasfcosta!

**Comment by meeber** [comment]
There's still quite a few references to `getFunctionName` in README, index.js, and test/index.js. I think we should go all-in with making it consistent throughout.

**Comment by lucasfcosta** [comment]
@meeber Well noticed!
Thank you guys for the review, you're awesome ❤️

**Comment by meeber** [comment]
I still see some in index and test/index :D

**Comment by lucasfcosta** [comment]
Thanks @meeber!
I really forgot those.
I think everything is okay now.

I'll start to work on the refactor of `check-error` and `chai` to use this as soon as it gets merged.

File: test/index.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 'use strict';
 var assert = require('simple-assert');
-var getFunctionName = require('..');
-describe('getFunctionName', function () {
-  it('getFunctionName', function () {
+var getFuncName = require('..');
+describe('getFuncName', function () {
+  it('getFuncName', function () {
     // Asserting that `getFunctionName` behaves correctly
```
Comment (meeber) [inline test/index.js]: @lucasfcosta Last one :D

Comment (lucasfcosta) [inline test/index.js]: @meeber you sir have good eyes, indeed.
Thanks!
Fixed!

**Comment by meeber** [comment]
LGTM

**Comment by vieiralucas** [comment]
I tough we were doing `retrieveTheNameOfTheProvidedFunctionNoMatterWhatTypeOfFunctionItIsOrWhatTheCurrentJavaScriptEnvironmentIs` 😄 

LGTM

**Pull request merged**